larq/metrics.py

```diff
@@ -71,7 +66,7 @@ def get_training_metrics():
 
 
 class LarqMetric(Metric):
```

Comment:
I think we can actually completely get rid of this class, since its only use is to add support for the `dtype ` argument in `add_weight` on TF 1.13.

Reply:
Nice; removed it in 6c9c691. Also checked and made sure that the signature and argument defaults still match [`tf.keras.metrics.Metric.add_weight()`](https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/keras/metrics.py#L245) and they do.